conjur/errors_messages.py

```diff
@@ -19,4 +19,4 @@
 MISMATCH_HOSTNAME_MESSAGE = "The machine's hostname did not match any names on the certificate. " \
                             "Make sure the names on the certificate (common name or SANs) match the machine's hostname."
 
-FETCH_CREDENTIALS_FAILURE_MESSAGE = "Failed to fetch credentials"
+FETCH_CREDENTIALS_FAILURE_MESSAGE = "Failed to fetch credentials. Log in again and try again."
```

Comment:
`Log in again and try again."`?

Reply:
@shulifink approval need here as well

Reply:
@shulifink reminder

Reply:
Log in again and re-run the command @shulifink